resources/site/how-to/workbench-application-platform/how-to-issue-custom-intent.md

```diff
@@ -0,0 +1,47 @@
+![SCION Workbench](/resources/site/logo/scion-workbench-banner.png)
+
+[Overview][menu-overview] | [Workbench][menu-workbench] | [Workbench&nbsp;Application&nbsp;Platform][menu-workbench-application-platform] | [Contributing][menu-contributing] | [Changelog][menu-changelog] | [Sponsoring][menu-sponsoring] | [Links][menu-links]
+|---|---|---|---|---|---|---|
+
+## Issuing a custom intent to the host application
+
+The workbench application platform provides an API to register [custom intent handlers](how-to-install-a-programmatic-intent-handler.md) in the host application. A child application can then use the `IntentService` to issue a generic intent to the platform. As those intents are generic, a client has to know what qualifier and payload he has to provide and what the response will be.
```

Comment:
I would rewrite the second sentence as follows:

... A child application can then use the `IntentService` to issue a respective intent by providing the intent's type and qualifier if any. Also, you can provide a payload if required by the other side.